### DISCUSSION TITLE
IntelliTouch / EasyTouch Emulator

### DISCUSSION THREAD

**Discussion by arrmo** [tagyoureit/nodejs-poolController#582] [General]
Hi,

OK, bit of an odd question - not all that new for me .. LOL. Out visiting a friend in CA, and he's looking to set up his (to be purchased) pool controller. I told him he has to use njsPC! Trying to get him set up, but he doesn't have the pool side of it yet ... the Linux box yes, but not pool controller.

Is there a way to emulate / simulate the pool side of things, so he can get up and running (with njsPC)? Or am I just (really) crazy? 😜 

Thanks!

**Comment by rstrouse** [comment]
If he just wants to try it out he can set up a Nixie controller to play around with.  It will pretend like there is a pool controller there.  When he gets his IntelliTouch, EasyTouch, or IntelliCenter it will then detect its existence and he will need to reconfigure it.  If the pool exists, but he doesn't have an OCP he may decide he doesn't need one.  

Personally, I would never buy any 20+ year old controller so if it is new he should be getting IntelliCenter.  IntelliTouch and EasyTouch are very long in the tooth.

**Reply by arrmo** [reply]
> Personally, I would never buy any 20+ year old controller so if it is new he should be getting IntelliCenter. IntelliTouch and EasyTouch are very long in the tooth.

Good point, thanks!



> If he just wants to try it out he can set up a Nixie controller to play around with.

Hmmm ... OK, not sure I understand that part - yet 😆. Nixie is the controller, or client side? Sorry, just not familiar with it.



> If the pool exists, but he doesn't have an OCP he may decide he doesn't need one.

Sorry, OCP?


Thanks!

**Reply by tagyoureit** [reply]
Since you've been gone, @rstrouse and I (mostly @rstrouse) have turned njsPC into a full fledge Pentair competitor.  Honestly, it eats easytouch for breakfast, intellitouch for lunch and the intellicenter for dinner.  It recently decided that it was still hungry and started devouring Aqualink as dessert.  See [Nixie setup](https://github.com/tagyoureit/nodejs-poolController/wiki/Nixie-Controller-Setup) and [Nixie installation](https://github.com/tagyoureit/nodejs-poolController/wiki/DIY-Standalone-Nixie-Pool-Controller).  Like you, I blew up my Easytouch a few months ago and Nixie completely replaced it and I couldn't be happier.  It's faster, better AND cheaper.

**Reply by arrmo** [reply]
Awesome - sounds great! Will do some digging, appreciate the links!

**Reply by rstrouse** [reply]
Just to clarify my lingo.  OCP stands for Outdoor Control Panel.  This is typically the board that is installed in the load center at the pad and in the Pentair case that is the motherboard + personality board for IntelliCenter, IntelliTouch, or EasyTouch.  Nixie will take on that role to control valves, circuit relays, RS485 equipment, schedules, and a whole lot more.

You can install njsPC, dashPanel, and REM on most platforms with an operating system that supports nodeJS including windows, linix, macOS, PiOS, and many others.  Even without pool equipment, Nixie will operate in emulation mode to allow you to play around with the software and witness the capabilities.   At any given time I have 3+ different pool controller instances emulating different pool setups.

**Reply by arrmo** [reply]
> Just to clarify my lingo. OCP stands for Outdoor Control Panel. This is typically the board that is installed in the load center at the pad and in the Pentair case that is the motherboard + personality board for IntelliCenter, IntelliTouch, or EasyTouch. Nixie will take on that role to control valves, circuit relays, RS485 equipment, schedules, and a whole lot more.

Seeing that, as I (we) dig into it - awesome! Very impressed, and I do think Nixie is the right answer. RPi very hard to get a hold of right now, so my friend bought a Libre Computer off of Amazon => hence this [question ](https://github.com/tagyoureit/nodejs-poolController/discussions/585)😜.



> You can install njsPC, dashPanel, and REM on most platforms with an operating system that supports nodeJS including windows, linix, macOS, PiOS, and many others. Even without pool equipment, Nixie will operate in emulation mode to allow you to play around with the software and witness the capabilities. At any given time I have 3+ different pool controller instances emulating different pool setups.

See that - we got it up and running on his Libre yesterday, very cool. Thanks!!
